src/components/Menu/Menu.module.css

```diff
@@ -0,0 +1,30 @@
+/*------------------------------------*\
+    # MENU
+\*------------------------------------*/
+
+/**
+ * Menu
+ */
+.menu {
+  position: relative;
+}
+
+.menu__popover {
+  max-width: var(--eds-l-max-width);
+  margin-top: var(--eds-size-1);
+  position: absolute;
+}
+
+.menu__button {
+  background-color: var(--light-color-button-background);
```

Comment:
Where's this CSS variable from?

Reply:
that's a GOOD question. I don't know why but some JS files with a bunch of variables popped up for me and I chose one from there. 😕 

The correct one (matches the variable in the figma) is `--eds-theme-color-form-background`